migration/v0.36/README.md

```diff
@@ -145,6 +145,12 @@ As with all scripts in this repository `CK8S_CONFIG_PATH` is expected to be set.
     ./migration/v0.36/apply/40-namespaces.sh execute
     ```
 
+1. Upgrade Velero
+
+    ```bash
+    ./migration/v0.36/apply/50-velero-crds-upgrade.sh execute
+    ```
+
 1. Upgrade applications:
 
     ```bash
```

Comment:
Below here we suggest these options to the operator:
```bash
    ./bin/ck8s apply {sc|wc}
    # or
    ./migration/v0.36/apply/80-apply.sh execut
``` 

but if they run `./bin/ck8s apply {sc|wc}` then they won't skip the apps specified [here](https://github.com/elastisys/compliantkubernetes-apps/blob/826dce05e61d40ad9ba8b6a47dd390ff841a5d10/migration/v0.36/apply/80-apply.sh#L11-L21)

Should we change this to only use the script `./migration/v0.36/apply/80-apply.sh execute`?

Reply:
I do not think it's an issue to run apply only, if no diff nothing should be applied
Let me know if you see this as a problem

Reply:
Okay, I did not try the `ck8s apply sc|wc` method, but if you think it's fine then sure.